include/zephyr/net/http/service.h

```diff
@@ -53,10 +55,12 @@ struct http_service_desc {
 	size_t backlog;
 	struct http_resource_desc *res_begin;
 	struct http_resource_desc *res_end;
+	const sec_tag_t *sec_tag_list;
+	size_t sec_tag_list_size;
```

Comment:
A (small) optimization if TLS isn't ever used, it to add compile guards.
```suggestion
#ifdef CONFIG_NET_SOCKETS_SOCKOPT_TLS
	const sec_tag_t *sec_tag_list;
	size_t sec_tag_list_size;
#endif
```